.github/workflows/cibuildwheel.yml

```diff
@@ -0,0 +1,113 @@
+name: Build CI wheels
+
+on:
+  push:
+    branches:
+      - master
+      - v[0-9]+.[0-9]+.x
+      - cibuildwheel
+    tags:
+      - v*
+
+jobs:
+  build_wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-18.04, windows-latest, macos-latest]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-python@v2
+        name: Install Python
+        with:
+          python-version: '3.7'
+
+      - name: Install cibuildwheel
+        run: |
+          python -m pip install cibuildwheel==1.6.3
+
+      - name: Build wheels for CPython 3.9 and Mac OS
+        if: matrix.os == 'macos-latest'
+        run: |
+          brew install libomp
+          python -m cibuildwheel --output-dir dist
+        env:
+          CIBW_BUILD: "cp39-*"
+          CIBW_MANYLINUX_X86_64_IMAGE: manylinux1
```

Comment:
what is this? Where does it get used?

Reply:
manylinux1 is the type of Linux wheel that gets produced (see my comment above). I guess [cibuildwheel](https://cibuildwheel.readthedocs.org) uses this env variable to determine how to build the wheels.

Reply:
It seems we probably do want to add more recent manylinux builds fairly soon. The following recent blog post may be useful: https://labs.quansight.org/blog/2020/11/manylinux1-is-obsolete-manylinux2010-is-almost-eol-what-is-next/